client-src/elements/form-definition.js

```diff
@@ -834,7 +834,23 @@ When in doubt, please check out https://goo.gle/devtools-checklist for details!
 const DEBUGGABILITY_SHIP_QUESTIONNAIRE = DEBUGGABILITY_ORIGIN_TRIAL_QUESTIONNAIRE;
 
 const TESTING_SHIP_QUESTIONNAIRE = (
-  'See http://go/chrome-wp-test-survey.'
+  html`<b>(1) Does your feature have sufficient automated test coverage (Unit tests, WPT, browser test  and other integration tests)?</b>
+Chrome requires at least 70% automation code coverage (<a href="https://analysis.chromium.org/coverage/p/chromium" target="_blank">dashboard</a>) running on the main/release branch and 70% Changelist <a href="https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/testing/code_coverage_in_gerrit.md" target="_blank">code coverage in Gerrit</a>?
+Do the automated tests have more than 93% green (flakiness < 7%) on CQ and CI builders?
+<ul>
+<li>Yes. My feature met the minimum automated test coverage and health requirements.
+<li>No. My feature does not meet the requirements since __________.
+</ul>
+<b>(2) How are performance tests conducted on Chromium builders?</b> List links to tests if any.
+
+<b>(3) Does this feature have non-automatable test cases that require manual testing? Do you have a plan to get them tested?</b>
+<ul>
+<li>No. All feature related test cases are automated.
+<li>Yes. There are non-automatable test cases and I have completed test execution or allocated resources to ensure the coverage of these test cases.
+<li>Yes. There are non-automatable test cases and my feature impacts Google products.
+</ul>
+<b>(4) If your feature impacts Google products, please fill in <a href="http://go/chrome-wp-test-survey">go/chrome-wp-test-survey</a>.</b> Make a copy, answer the survey questions, and provide a link to your document here.
```

Comment:
```suggestion
<b>(4) If your feature impacts Google products, please complete the survey at <a href="http://go/chrome-wp-test-survey">go/chrome-wp-test-survey</a>.</b> Make a copy, answer the survey questions, and provide a link to your document here.
```

Reply:
Good suggestion, but I think the word "survey" is already used twice in this item.  And, the overall block of text is labeled "survey questions".  It just seems to repetitive.